docs/source/quickstart_mlops.rst

```diff
@@ -0,0 +1,160 @@
+.. _quickstart-mlops:
+
+Quickstart: Compare runs, choose a model, and deploy it to a REST API
+======================================================================
+
+
+In this quickstart, you will:
+
+- Run a hyperparameter sweep on a training script
+- Compare the results of the runs in the MLflow UI
+- Choose the best run and register it as a model
+- Deploy the model to a REST API
+- Build a container image suitable for deployment to a cloud platform
+
+As an ML Engineer or MLOps professional, you can use MLflow to compare, share, and deploy the best models produced by the team. In this quickstart, you will use the MLflow Tracking UI to compare the results of a hyperparameter sweep, choose the best run, and register it as a model. Then, you will deploy the model to a REST API. Finally, you will create a Docker container image suitable for deployment to a cloud platform.
+
+.. image:: _static/images/quickstart/quickstart_tracking_overview.png
+    :width: 800px
+    :align: center
+    :alt: Diagram showing Data Science and MLOps workflow with MLflow
+
+
+Set up
+------
+
+- Install MLflow. See the `MLflow Data Scientist quickstart <quickstart>`_ for instructions
```

Comment:
This link didn't work for me.

Reply:
Yoiks. OK, this is the wrong pattern. I searched for it and saw that I'd repeated the mistake in *what-is-mlflow* too. Fixed both this and that in 5dce44aaf.